apps/docs/src/app/core/component-docs/content-density-2/content-density-user-2/content-density-user.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Inject, ViewEncapsulation } from '@angular/core';
+import { DestroyedService } from '@fundamental-ngx/core/utils';
+import {
+    ContentDensityConsumer,
+    contentDensityConsumer,
+    ContentDensityMode
+} from '@fundamental-ngx/core/content-density';
+
+@Component({
+    selector: 'fundamental-ngx-content-density-user-two',
+    template: '<ng-content></ng-content> - {{ _contentDensityConsumer | async }}',
+    styleUrls: ['./content-density-user.component.scss'],
+    host: {
+        class: 'example-component'
+    },
+    providers: [
+        DestroyedService,
+        contentDensityConsumer({
+            modifiers: {
+                [ContentDensityMode.COMPACT]: 'example-component--compact',
+                [ContentDensityMode.CONDENSED]: 'example-component--condensed'
+            },
+            defaultContentDensity: ContentDensityMode.COMPACT
+        })
+    ],
+    encapsulation: ViewEncapsulation.None
+})
+export class ContentDensityUserTwoComponent {
+    constructor(@Inject(ContentDensityConsumer) readonly _contentDensityConsumer: ContentDensityConsumer) {}
```

Comment:
i would recommend to notice that without explicitly specifying in the constructor consumer instance won't be created. kinda obvious thing but some developers may don' know this.

Reply:
I would say Examples are not ready yet with their docs... I should probably ask @ynnenu for help